index.html

```diff
@@ -110,16 +126,17 @@ <h2 class="work-header">Multi-Post
             </ul>
             <button class="btn-project">See Project</button>
```

Comment:
- [ ] [REQUIRED] Good job so far! Please give correct style to all buttons as shown in [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122&t=Uy7TAUKVJbGSY3Ol-0) 🙏🏼 

![image](https://user-images.githubusercontent.com/63915024/213466821-45fbc50c-ae62-48b3-9c32-bf23a54bf3a3.png)